.nvmrc

```diff
@@ -0,0 +1 @@
+14.19.0
```

Comment:
Let's just make it "14" because the minor versions shouldn't matter much for functionality and we shouldn't force travis and engineers to install a specific version (which may also become vulnerable one day)